Dashboard/extract_combined.py

```diff
@@ -49,7 +51,7 @@ def extract_product_information(conn: connection, product_url: str) -> tuple:
         website_data = EXTRACT_FUNCTIONS[store_name](product_url)
         website_data['website_id'] = website_id
         return website_data
-    except ValueError:
+    except Exception:
```

Comment:
Why using a less specific error here?

Reply:
That's a good spot, thank you! I'll change that now!